NomadUI/Widgets/GenericPluginEditor.cpp

```
@@ -0,0 +1,376 @@
+}
+
+GenericPluginEditor::~GenericPluginEditor() {
+}
+
+void GenericPluginEditor::buildParameterWidgets() {
+    if (!m_instance) return;
+    
+    m_parameters.clear();
+    
+    auto allParams = m_instance->getParameters();
+    printf("[GenericPluginEditor] Plugin has %zu total parameters\n", allParams.size());
```

Cppcheck (2.19.0)

[style] The function 'setOnClose' is never used.

(unusedFunction)